packages/state-manager/src/utils/tests/factories.ts

```diff
@@ -205,6 +206,14 @@ export const getFactory = async (store: Store) => {
         store.dispatch(messagesActions.incomingMessages({
           messages: [payload.message]
         }))
+        if (payload.message.type === MessageType.File || MessageType.Image) {
```

Comment:
What is that for? It should be saga's responsibilities to set proper download status for file (on incoming messages action)